src/tools/management/commands/update_metrics.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
```

Comment:
Doing just one day at a time makes sense, but we still need the table to include the aggregations of all log events from the beginning (the way output_user_metrics does). Could be a one-time data migration?

Reply:
Manually run on server with a defined date range.